src/cata_tiles.cpp

```diff
@@ -680,34 +680,44 @@ void cata_tiles::load_ascii_set( JsonObject &entry, int offset, int size )
 
 void cata_tiles::load_tilejson_from_file( JsonObject &config, int offset, int size )
 {
-    if( !config.has_member( "tiles" ) ) {
+    if ( !config.has_member( "tiles" ) ) {
         config.throw_error( "\"tiles\" section missing" );
     }
 
     JsonArray tiles = config.get_array( "tiles" );
-    while( tiles.has_more() ) {
+    while ( tiles.has_more() ) {
         JsonObject entry = tiles.next_object();
 
-        std::string t_id = entry.get_string( "id" );
+        std::vector<std::string> ids;
+        if ( entry.has_string( "id" ) ) {
+            ids.push_back( entry.get_string( "id" ) );
+        } else if ( entry.has_array( "id" ) ) {
+        JsonArray ida = entry.get_array( "id" );
+            while ( ida.has_more() ) {
+                ids.push_back( ida.next_string() );
+            }
+        }
+        for ( auto t_id : ids ) {
```

Comment:
get_tags returns a set which is sorted by value. I want to preserve the order they are listed in the file, so it will be easier to debug sprite loading problems in the future.